### DISCUSSION TITLE
Despite of installing Hiddify panel, I can’t connect through my proxies چرا با وجود نصب پنل هایدیفای کانکشن‌های من وصل نمیشن

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#301] [Q&A سوالات رایج] [answered]
چرا با وجود نصب پنل هایدیفای کانکشن‌های من وصل نمی‌شوند؟

یا داستان چیه که با رایتل وصل میشه ولی مثلا با ایرانسل وصل نمیشه

**Comment by lymanjre** [comment, marked as answer]
همانطور که در مقدمات نصب پیکربندی پنل گفته شده، این موضوع چند دلیل بیشتر ندارد:

> ۱. آیپی یا دامنه بلاک شده است. 
> ۲. آیپی نیمه‌بلاک و در لیست زرد قرار گرفته شده است. مثلا در بعضی ISPها وصل و در بعضی وصل نیست. 
> ۳. اگر از دامین CDN استفاده شده است، آیپی کلادفلر تمیزی انتخاب نشده است. 
> 

بنابراین این پنل قرار نیست معجزه بکند. نصب آن مقدماتی نیاز دارد. حتما قبل از پیکربندی پنل، مطمئن شوید آیپی و یا دامنه شما سالم است.[نحوه تست](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%D8%A7%D8%B7%D9%85%DB%8C%D9%86%D8%A7%D9%86-%D8%A7%D8%B2-%D8%AA%D9%85%DB%8C%D8%B2-%D8%A8%D9%88%D8%AF%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%DB%8C%D8%A7-%D8%AF%D8%A7%D9%85%D9%86%D9%87-%D8%B3%D8%B1%D9%88%D8%B1) 


اگر از دامنه CDN استفاده می‌شود نیاز است [آیپی تمیز کلادفلر](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%DB%8C%D8%A7%D9%81%D8%AA%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%D8%AA%D9%85%DB%8C%D8%B2-%DA%A9%D9%84%D8%A7%D8%AF%D9%81%D9%84%D8%B1)  پیدا کنید و در فیلد اجباری دامنه در منوی دامنه‌ها قرار دهید. 


اگر شما توانسته اید از یک ابزار دیگه، روی همون سرور و همون دامنه، متصل بشید، بلافاصله اون لینک تروجان یا وی لس یا وی مس را برای ما بفرستید تا توی پنل اضافه کنیم




# راه کارهایی برای حل مشکل
- آی پی سالم پیدا کنید
- دامنه ی جدیدی استفاده کنید
- میتوانید از CDN استفاده کنید
- [میتوانید از Relay استفاده کنید](https://github.com/hiddify/hiddify-config/discussions/129)
- از دامنه ی Fake استفاده کنید


# لینک مطالب مرتبط برای این راه‌کار

> [۱. نحوه اطمینان از سالم بودن آیپی یا دامنه سرور](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%D8%A7%D8%B7%D9%85%DB%8C%D9%86%D8%A7%D9%86-%D8%A7%D8%B2-%D8%AA%D9%85%DB%8C%D8%B2-%D8%A8%D9%88%D8%AF%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%DB%8C%D8%A7-%D8%AF%D8%A7%D9%85%D9%86%D9%87-%D8%B3%D8%B1%D9%88%D8%B1)
> [۲. نحوه یافتن آیپی تمیز کلادفلر](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%DB%8C%D8%A7%D9%81%D8%AA%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%D8%AA%D9%85%DB%8C%D8%B2-%DA%A9%D9%84%D8%A7%D8%AF%D9%81%D9%84%D8%B1) 
 > [۳. انواع دامنه و نحوه ثبت آن‌ها](https://github.com/lumanjre/hiddify-config/wiki/%D8%A7%D9%86%D9%88%D8%A7%D8%B9-%D8%AF%D8%A7%D9%85%D9%86%D9%87-%D9%88-%D9%86%D8%AD%D9%88%D9%87-%D8%AB%D8%A8%D8%AA-%E2%80%8C%D8%A2%D9%86%E2%80%8C%D9%87%D8%A7)

**Comment by lymanjre** [comment]
# لینک مطالب مرتبط برای این راه‌کار

> [۱. نحوه اطمینان از سالم بودن آیپی یا دامنه سرور](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%D8%A7%D8%B7%D9%85%DB%8C%D9%86%D8%A7%D9%86-%D8%A7%D8%B2-%D8%AA%D9%85%DB%8C%D8%B2-%D8%A8%D9%88%D8%AF%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%DB%8C%D8%A7-%D8%AF%D8%A7%D9%85%D9%86%D9%87-%D8%B3%D8%B1%D9%88%D8%B1)
> [۲. نحوه یافتن آیپی تمیز کلادفلر](https://github.com/lumanjre/hiddify-config/wiki/%D9%86%D8%AD%D9%88%D9%87-%DB%8C%D8%A7%D9%81%D8%AA%D9%86-%D8%A2%DB%8C%D9%BE%DB%8C-%D8%AA%D9%85%DB%8C%D8%B2-%DA%A9%D9%84%D8%A7%D8%AF%D9%81%D9%84%D8%B1) 
 > [۳. انواع دامنه و نحوه ثبت آن‌ها](https://github.com/lumanjre/hiddify-config/wiki/%D8%A7%D9%86%D9%88%D8%A7%D8%B9-%D8%AF%D8%A7%D9%85%D9%86%D9%87-%D9%88-%D9%86%D8%AD%D9%88%D9%87-%D8%AB%D8%A8%D8%AA-%E2%80%8C%D8%A2%D9%86%E2%80%8C%D9%87%D8%A7)

**Comment by sdimitt** [comment]
امروز ناگهانی همه‌ی لینکهای من از کار افتادند. اما بعد از ریستارت کردن سرور مشکل حل شد. لازم به ذکره که ریستارت از داخل اپ hiddify مشکل را حل نکرد، مستقیما vm را ریستارت کردم تا مشکل حل شد.

**Comment by Farazzzssss** [comment]
shoma yebaram warp ro off bokon test bokon
